### PR TITLE
Revert persisting mountedBlockDeviceList

### PR DESCRIPTION
**Pull request by bostko**
I tested:
- OpenstackV2ExtraHddBodyEffectorLiveTest pass
- AwsExtraHddBodyEffectorLiveTest they pass
- OpenStackNewVolumeCustomizerLiveTest pass

### CODE REVIEW

File: blockstore/src/main/java/brooklyn/location/blockstore/effectors/ExtraHddBodyEffector.java [hunk 1]
```diff
@@ -57,8 +57,8 @@
```
Comment (iyovcheva) [inline blockstore/src/main/java/brooklyn/location/blockstore/effectors/ExtraHddBodyEffector.java]: Can you rename to `ExtraVolumesEffector`?

File: blockstore/src/main/java/brooklyn/location/blockstore/NewVolumeCustomizer.java [hunk 2]
```diff
@@ -105,7 +93,6 @@ protected VolumeManager getVolumeManager(JcloudsMachineLocation machine) {
                 throw new UnsupportedOperationException("Tried to invoke addExtraHdd effector on entity " + getCallerContext(machine) + " for cloud "
                         + provider + " which does not support adding disks from an effector.");
```
Comment (bostko) [inline blockstore/src/main/java/brooklyn/location/blockstore/NewVolumeCustomizer.java]: I think the message is sensible isn't it?

File: blockstore/src/main/java/brooklyn/location/blockstore/effectors/ExtraHddBodyEffector.java (hunk 1 above)
Comment (aledsage) [inline blockstore/src/main/java/brooklyn/location/blockstore/effectors/ExtraHddBodyEffector.java]: Given how close we are to release, let's leave the classname as-is. We can rename in a future release, handling it with our `deserializingClassRenames.properties` stuff.

**Comment by aledsage** [comment]
LGTM - merging.

@bostko @iyovcheva have you tested this on vCloud Director?

**Pull request merged**